### PR TITLE
Fix Internal logging

### LINKED ISSUE #74: Internal Logging
Surround Internal Logging does not work with Web Endpoints

### LINKED ISSUE #77: Print Statements
Replace `Print` in api.py with logs once #74 gets fixed.

### PR DESCRIPTION
**Pull request by AkshatBajaj**
Fixes #74 
Fixes #77

### CODE REVIEW

File: surround/cli.py [hunk 1]
```diff
@@ -266,7 +269,7 @@ def parse_tool_args(parsed_args, remote_parser, tool):
         parse_init_args(parsed_args)
 
 def main():
-    logging.disable(logging.CRITICAL)
+    # logging.disable(logging.CRITICAL)
```
Comment (ScottyB) [inline surround/cli.py]: Can you remove this comment if it isn't needed?

File: surround/cli.py [hunk 2]
```diff
@@ -166,7 +168,8 @@ def parse_run_args(args):
             "files" : [
                 ("{project_name}/config.yaml", "output:\n  text: Hello World"),
                 ("dodo.py", "")
-            ]
+            ],
+            "templates" : []
```
Comment (ScottyB) [inline surround/cli.py]: Why is this needed?

File: surround/cli.py (hunk 1 above)
Comment (AkshatBajaj) [reply surround/cli.py]: Done!

File: surround/cli.py (hunk 2 above)
Comment (AkshatBajaj) [reply surround/cli.py]: Because of this: https://github.com/dstil/surround/blob/b5e6f46825917093d3627adfa84b0eca2cc01fc3/surround/linter.py#L37

Comment (AkshatBajaj) [reply surround/cli.py]: Otherwise, it will throw key not found error!
